R/tax_expand_lat.R

```diff
@@ -0,0 +1,98 @@
+#' Generate pseudo-occurrences from latitudinal range data
+#'
+#' A function to generate pseudo-occurrences for taxa based on latitudinal
+#' ranges (e.g. the output of the 'lat' method in \code{\link{tax_range_geo}}).
+#' While the resulting pseudo-occurrences should not be treated as equivalent
+#' to actual occurrence data (e.g. like that from the Paleobiology Database),
+#' such pseudo-occurrences may be useful for  performing statistical analyses
+#' where the row representing a taxon must be replicated for each latitudinal
+#' bin through which the taxon ranges.
+#'
```

Comment:
All this seems fine - really glad you included a sentence about how these won't equate to full occurrences.